lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1317,19 +1328,15 @@ def main():
             esxi_hostname=dict(type='str'),
             cluster=dict(type='str'),
             wait_for_ip_address=dict(type='bool', default=False),
-            snapshot_src=dict(type='str', default=None),
+            snapshot_src=dict(type='str'),
             linked_clone=dict(type='bool', default=False),
             networks=dict(type='list', default=[]),
             resource_pool=dict(type='str'),
-            customization=dict(type='dict', no_log=True, default={}),
+            customization=dict(type='dict', default={}, no_log=True),
         ),
         supports_check_mode=True,
         mutually_exclusive=[
-            ['esxi_hostname', 'cluster'],
-        ],
-        required_together=[
-            ['state', 'force'],
-            ['template'],
```

Comment:
Can someone explain what the purpose is of this ?
Both `state` and `force` have a default value, so they are always set anyway.